### PR TITLE
fix(image): switch to adoptopenjdk/openjdk11:alpine-jre

### PR DESCRIPTION
**Pull request by onedr0p**
You may want to test this as I didn't have a chance to build Doplarr and test it.

### CODE REVIEW

**Comment by kiranshila** [comment]
Adoptopenjdk (my preferred JDK) has JRE builds for alpine that are half the side of the openjdk:11 image at 50MB

`adoptopenjdk/openjdk11:alpine-jre`

**Comment by kiranshila** [comment]
But  switching to alpine means I need to understand all the apt stuff you did haha

**Comment by onedr0p** [comment]
I can do it in Alpine. I'll update this PR.

**Comment by onedr0p** [comment]
So you also want doplarr to be built in the image? There's methods to do this that can still keep a light image.

**Comment by kiranshila** [comment]
> Do you also want doplarr to be built in the image? There's methods to do this that can still keep a light image.

I don't think we need to. Github CI is building it and copying the JAR in and I don't think I need to debug in the image anymore.

**Comment by onedr0p** [comment]
@kiranshila updated, please verify :)

File: Dockerfile [hunk 1]
```diff
@@ -1,31 +1,15 @@
...
-  DEBCONF_NONINTERACTIVE_SEEN=true \
-  DEBIAN_FRONTEND="noninteractive"
+FROM adoptopenjdk/openjdk11:alpine-jre
 
 WORKDIR /app
 
 RUN \
-  apt-get -qq update \
-  && apt-get install -qy \
+  apk add --no-cache
```
Comment (kiranshila) [inline Dockerfile:6]: This line needs a `\`

**Review by kiranshila — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (onedr0p) [reply Dockerfile]: that's what I get for doing it LIVE

Comment (onedr0p) [reply Dockerfile]: Updated.

**Review by kiranshila — APPROVED** [review]

**Comment by kiranshila** [comment]
Thanks! Looks great!

**Pull request merged**